src/main/java/com/amplitude/api/AmplitudeClient.java

```diff
@@ -1161,6 +1171,9 @@ protected long logEvent(String eventType, JSONObject eventProperties, JSONObject
             if (appliedTrackingOptions.shouldTrackAdid() && deviceInfo.getAdvertisingId() != null) {
                 apiProperties.put("androidADID", deviceInfo.getAdvertisingId());
             }
+            if (appliedTrackingOptions.shouldTrackAppSetId() && deviceInfo.getAppSetId() != null) {
+                apiProperties.put("androidAppSetId", deviceInfo.getAppSetId());
```

Comment:
`android_app_set_id`

Reply:
?

Reply:
@haoliu-amp this is per the discussion earlier, Kurt mentioned the attributes in backend is mostly with underscore, so just change the label to align it.